ui/plugins/controller.js

```diff
@@ -381,6 +380,7 @@ treeherder.controller('PluginCtrl', [
                 return;
             }
 
+            // TODO: Figure out if this conditional is needed any more
```

Comment:
Perhaps the thing to do with it now is pop up an alert saying something like "no system exists to handle this action for this job type" if the conditional is false?

Reply:
Good idea - I've added a:
`thNotify.send('Unable to backfill this job type!', 'danger', { sticky: true });`